airbyte-integrations/connectors/source-chargify/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Your connector does not implement incremental sync, please remove this file.

Reply:
Done!